.github/workflows/miscellaneous-integration.yml

```diff
@@ -75,6 +74,12 @@
         - name: Clean Install
           if:  steps.test_needed.outputs.integ_test_required == 'true'
           run: npm ci
+        - name: Run Data Message Integration Test
+          if:  steps.test_needed.outputs.integ_test_required == 'true'
+          run: npm run test:integration-data-message
         - name: Run Meeting Readiness Checker Integration Test
           if:  steps.test_needed.outputs.integ_test_required == 'true'
-          run: npm run test:integration-meeting-readiness-checker
\ No newline at end of file
+          run: npm run test:integration-meeting-readiness-checker
+        - name: Run Meeting End Integration Test
```

Comment:
Can we also add messaging session at the end ?

Reply:
Talked offline about this, ignoring this comment.